src/gui/log/logmodel.cpp

```diff
@@ -134,6 +138,12 @@ void BaseLogModel::reset()
 
 LogMessageModel::LogMessageModel(QObject *parent)
     : BaseLogModel(parent)
+    , m_foregroundForMessageTypes {
+        {Log::NORMAL, UIThemeManager::instance()->getColor(QLatin1String("Log.Normal"), QApplication::palette().color(QPalette::WindowText))},
+        {Log::INFO, UIThemeManager::instance()->getColor(QLatin1String("Log.Info"), QColor(0, 0, 255))}, // blue
```

Comment:
please use `{0, 0, 255}` if that is possible.

Reply:
should I replace these with named constants?

Reply:
there was no constant for orange so I didn't change any of them

Reply:
>should I replace these with named constants?

If there are matching constants from Qt, why not?

Do you mind dropping the `QColor` word too? I suppose just `{0, 0, 255}` would work.

Reply:
> If there are matching constants from Qt, why not?

do you mean by - wherever possible replace with constant and if no constant use `{0, 0, 255}`?

also personally speaking `{0, 0, 255}` will look confusing,

Reply:
>do you mean by - wherever possible replace with constant and if no constant use {0, 0, 255}?

Yes.

>also personally speaking {0, 0, 255} will look confusing,

If you feel so, then you can keep the `QColor`.